### PR TITLE
Avatar and badge

### PR DESCRIPTION
**Pull request by Arpit-dotcom**
avatar and badge components are created

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    color: rgb(36, 36, 235);
+}
+.fa-youtube{
+    color: red;
+}
+.fa-facebook-messenger{
+    color: rgb(14, 14, 204);
+}
+.fa-whatsapp{
+    color: green;
```
Comment (anupkgurung) [inline style.css:59]: colors can me moved to root and use variable instead

**Review by anupkgurung — COMMENTED** [review]
color could be added to root.

**Review by Creator-singh — APPROVED** [review]
very nice🚀 i liked the use of rem throughout the code specially

File: style.css (hunk 1 above)
Comment (Arpit-dotcom) [reply style.css]: Done sir

**Pull request merged**